src/worker/predictive-text-studio-worker-impl.ts

```diff
@@ -39,4 +50,69 @@ export class PredictiveTextStudioWorkerImpl
   updateBCP47Tag(bcp47Tag: string): Promise<void> {
     return this.storage.updateBCP47Tag(bcp47Tag);
   }
+
+  async generateKMPFromStorage(): Promise<Blob | void> {
+    // TODO: Parse multiple dictionary sources, right now just reading the first file
+    this.onPackageCompileStart((): void => {
+      console.log("Compile starts.");
+    });
```

Comment:
Call the registered callback instead:

```suggestion
    this._emitPackageCompileStart();
```